quartz/plugins/emitters/contentIndex/indexTree.ts

```diff
@@ -0,0 +1,142 @@
+import { ContentDetails } from "../contentIndex"
```

Comment:
do we need to create an additional file here? if this is purely utils then it should live under `quartz/util`, otherwise just leave all content in contextIndex.ts to reduce switch overhead.

This is mainly stylistic preferences, I always prefer working with one big file :)